.github/workflows/publish-wiki.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Wiki
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Upload Documentation to Wiki
+        uses: SwiftDocOrg/github-wiki-publish-action@v1
+        with:
+          path: "wiki"
+        env:
+          GH_PERSONAL_ACCESS_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```

Comment:
The implementation here looks good but I think it'd be a good idea for us to use a GitHub app rather than a token directly. Maybe we could revise the perms for the existing DX app?